modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -16,12 +16,32 @@ As the fields in the `kubeletConfig` object are passed directly to the kubelet f
 
 Consider the following guidance:
 
-* Create one `KubeletConfig` CR for each machine config pool with all the config changes you want for that pool. If you are applying the same content to all of the pools, you need only one `KubeletConfig` CR for all of the pools.
-
 * Edit an existing `KubeletConfig` CR to modify existing settings or add new settings, instead of creating a CR for each change. It is recommended that you create a CR only to modify a different machine config pool, or for changes that are intended to be temporary, so that you can revert the changes.
 
+* Create one `KubeletConfig` CR for each machine config pool with all the config changes you want for that pool. If you are applying the same content to all of the pools, you need only one `KubeletConfig` CR for all of the pools.
```

Comment:
So I know this is just a change of ordering but I think this may be incorrect from before. You should have a separate kubeletconfig for each pool, or there could be unintended errors.